tests/Feature/Api/Projects/Applications/ViewLogTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Tests\Feature\Api\Sites;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Http\Response;
+use Servidor\Projects\Application;
+use Servidor\Projects\Project;
+use Tests\RequiresAuth;
+use Tests\TestCase;
+
+class ViewLogTest extends TestCase
+{
+    use RefreshDatabase;
+    use RequiresAuth;
+
+    /** @test */
+    public function can_retrieve_log_file(): void
+    {
+        $app = new Application(['template' => 'php']);
+        $project = Project::create(['name' => 'logtest']);
+        $project->applications()->save($app);
+
+        $response = $this->authed()->getJson("/api/projects/{$project->id}/logs/php.app-{$app->id}.log");
+
+        $response->assertOk();
+    }
+
+    /** @test */
+    public function cannot_get_log_if_project_id_does_not_match(): void
+    {
```

Comment:
For some reason this is just chucking out a 'Server Error' exception rather than the 'Exception' we expect.

```suggestion
    {
        $this->withoutExceptionHandling();
```